app/models/translation.rb

```diff
@@ -10,6 +10,20 @@ class Translation < ActiveRecord::Base
 
   @@root_folder_name = 'public/srt/'
 
+
+  @@STATUS_HASH = {0 => :incomplete, 1 => :complete, 2 => :complete_with_priority}
+  @@POINTS_HASH = {:incomplete => 0, :complete => 1, :complete_with_priority => 2}
+  @@COMPLETE_POINTS = 1
+  @@COMPLETE_WITH_PRIORITY_POINTS = 2
+
+  def points
+    @@POINTS_HASH[self.status]
+  end
+
+  def status
```

Comment:
Big no-no. You're overriding an attribute accessor, which makes this method endlessly recursive. Should probably rename this `status_symbol`.